app/graphql/re_sip_e_be_schema.rb

```diff
@@ -37,4 +37,9 @@ def self.object_from_id(global_id, query_ctx)
     # For example, use Rails' GlobalID library (https://github.com/rails/globalid):
     GlobalID.find(global_id)
   end
+
+  rescue_from(ActiveRecord::RecordNotFound) do |err, obj, args, ctx, field|
+    raise GraphQL::ExecutionError, "#{field.type.unwrap.graphql_name} not found"
```

Comment:
Glad you found info on this `rescue_from` method available for GraphQL it will make Sad Path testing easy for all our DB find methods